src/libstd/collections/hash/map.rs

```diff
@@ -1585,7 +1585,7 @@ mod test_map {
     use super::Entry::{Occupied, Vacant};
     use iter::{range_inclusive, range_step_inclusive, repeat};
     use cell::RefCell;
-    use rand::{weak_rng, Rng};
```

Comment:
As these are used for tests and `XorShiftRng` needs to be seeded manually now, I think preserved noise-to-signal ratio is much more valuable than those few µs we’d save by not using thread_rng.

`strconv` benchmarks are another story, though. Will update these tomorrow.